commown_self_troubleshooting/i18n/fr.po

```diff
@@ -969,21 +1052,18 @@ msgid "Fairphone 3/3+"
 msgstr ""
 
 #. module: commown_self_troubleshooting
+#: model:commown_self_troubleshooting.category,name:commown_self_troubleshooting.fp4-ts-cat
 #: model:ir.ui.view,website_meta_title:commown_self_troubleshooting.fp4-screen
 #: model:website.page,website_meta_title:commown_self_troubleshooting.fp4-screen-page
 msgid "Fairphone 4"
 msgstr ""
 
-#. module: commown_self_troubleshooting
-#: model_terms:ir.ui.view,arch_db:commown_self_troubleshooting.fp2-micro
-#: model_terms:website.page,arch_db:commown_self_troubleshooting.fp2-micro-page
-msgid "Fonctionnel"
-msgstr ""
-
 #. module: commown_self_troubleshooting
 #: model:project.task.type,description:commown_self_troubleshooting.stage_callback
 msgid "For cases where the customer has not responded to the ticket once, which suggests that he/she is not receiving the emails!  "
-msgstr "Pour les cas où le client n'a pas répondu une seule fois au ticket, ce qui laisse soupçonner qu'il/elle ne reçoit pas les mails !  "
```

Comment:
le/la client.e

Reply:
Je parlais ici de la traduction de "For cases where the customer has not responded" (la traduction emploie l'écriture inclusive dans la moitié de la phrase seulement). Ça reste à faire donc.

Reply:
Bon en fait je constate que cette traduction est débile mais est là parce que : 
1. l'étape stage_callback est définie dans commown_self_troubleshooting donc odoo extrait tous ses attributs traduits
2. j'imagine que dans Odoo il y a une vieille version originale de la description de l'étape, en anglais, complètement HS maintenant.

Je suggère donc de modifier sur ton instance et en prod la source de la traduction (fr et de tant qu'à faire) en y copiant la version fr, et de ré-exporter une dernière fois le fr.po.

Prends le temps de te poser ces questions : quand on te demande un truc, demande-toi toujours si c'est pertinent en prenant un poil de recul, quelle que soit la personne qui te le demande (même si c'était ton N+4 dans une grosse boite ;-) ).

Reply:
fr.po et de.po envoyés sur le ticket associé @fcayre